build.gradle.kts

```diff
@@ -27,7 +27,7 @@ dependencies {
     }
 
     implementation("com.uchuhimo", "konf", "0.22.1")
-    implementation("net.rcarz", "jira-client", "0.6.3-IHTSDO")
+    implementation("com.github.rcarz", "jira-client", "master-SNAPSHOT")
```

Comment:
@urielsalis, shouldn't this use a specific version (or commit hash)? Otherwise that might cause unexpected build failures in the future.